front_end/webserver.py

```diff
@@ -185,9 +188,12 @@ def post(self, course):
                     if re.search(r"[^\w ]", title):
                         result = "Error: The title can only contain alphanumeric characters and spaces."
                     else:
-                        content.specify_course_basics(course_basics, title, visible)
-                        content.specify_course_details(course_details, introduction, None, datetime.datetime.now())
-                        course = content.save_course(course_basics, course_details)
+                        if len(title) > 20:
+                            result = "Error: Title cannot exceed 20 characters."
```

Comment:
Please change this to "Error: The title cannot exceed..." That way it uses valid grammar and is consistent with the error message above.